### PR TITLE
CodePane element

### PR DESCRIPTION
**Pull request by bmathews**

### CODE REVIEW

File: src/components/syntax.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    el.type = 'text/css';
+    el.id = STYLE_TAG_ID;
+    const cssText = '.spectacle-viewer-syntax code { font-family: inherit; }';
+    if (el.styleSheet) {
+      el.styleSheet.cssText = cssText;
+    } else {
+      el.appendChild(document.createTextNode(cssText));
+    }
+    document.head.appendChild(el);
+  }
```
Comment (bmathews) [inline src/components/syntax.js:71]: No way to override the font without this at the moment.

https://github.com/conorhastings/react-syntax-highlighter/issues/19

File: src/components/syntax.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  tomorrow,
+  tomorrowNight,
+};
+
+const STYLE_TAG_ID = '_spectacle-viewer-syntax-style';
+
+class Syntax extends Component {
+
+  componentDidMount() {
+    if (document.getElementById(STYLE_TAG_ID)) return;
```
Comment (rgerstenberger) [inline src/components/syntax.js]: code style nit, spectacle-editor lint has no single line if statements.

**Review by rgerstenberger — APPROVED** [review]
Minor code style nit, otherwise LGTM

File: src/components/syntax.js (hunk 2 above)
Comment (bmathews) [reply src/components/syntax.js]: Updated. It'd be nice if they used the same lint config!

**Pull request merged**